app/src/main/java/io/apicurio/registry/ccompat/store/CCompatConfig.java

```diff
@@ -36,4 +36,8 @@ public class CCompatConfig {
     @Info(category = "ccompat", description = "Legacy ID mode (compatibility API)", availableSince = "2.0.2.Final")
     Supplier<Boolean> legacyIdModeEnabled;
 
+    @Dynamic(label = "Canonical hash mode (compatibility API)", description = "When selected, the Schema Registy compatibility API uses the canonical hash instead of the regular hash of the content.")
+    @ConfigProperty(name = "registry.ccompat.use-canonical-hash", defaultValue = "false")
+    @Info(category = "ccompat", description = "Canonical hash mode (compatibility API)", availableSince = "2.2.6.Final")
```

Comment:
The next upstream release will be 2.3.0.Final.  I'll change this. :)

Reply:
Agh, I almost guessed it :)